test/src/test.c

```diff
@@ -9,10 +10,12 @@
 #include "pos.h"
 #include "tiny.h"
 
+#ifdef TINY_COMPILER
+
 static int MallocCalls = 0;
 static int FreeCalls = 0;
 
-static void *Alloc(void *ptr, size_t size, void *userdata) {
+static void* Alloc(void *ptr, size_t size, void *userdata) {
```

Comment:
What formatter did you use for this PR? I think we want to stick to `clang-format` for this repo.

Reply:
I use Eclipse for programming.
If you give me a complete definition of the format I can configure it.

Reply:
@hiperiondev there is a `.clang-format` at the root of the repo that dictates the clang format style guidelines used for this project. `clang-format` should automatically pick it up when run anywhere in the subtree.

Reply:
Done.